app/src/main/java/com/fstyle/androidtrainning/screen/main/mainfragments/mysong/subfragment/album/AlbumFragment.java

```diff
@@ -3,17 +3,27 @@
 import android.os.Bundle;
 import android.support.annotation.NonNull;
 import android.support.v4.app.Fragment;
+import android.support.v7.widget.GridLayoutManager;
+import android.support.v7.widget.LinearLayoutManager;
```

Comment:
ERROR: Unused import: android.support.v7.widget.LinearLayoutManager. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports